app/graphql/mutations/profile/update.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class Mutations::Profile::Update < Mutations::Base
+  include FancyMutation
+
+  description 'Update profile'
+
+  input do
+    argument :id, ID,
+      required: true,
+      description: 'Your ID or the one of another user.'
```

Comment:
I think let's make this optional, it's only useful to admins, and the default can just be `current_user`